spec/generated/bundled-polaris-catalog-service.yaml

```diff
@@ -635,7 +635,7 @@ paths:
                 TableToUpdateDoesNotExist:
                   $ref: '#/components/examples/NoSuchTableError'
         '409':
-          description: Conflict - CommitFailedException, one or more requirements failed. The client may retry.
+          description: Conflict - CommitConflictException, one or more requirements failed. The client may retry.
```

Comment:
I'd prefer to avoid mentioning rather internal Polaris exception type names in the API doc.

Reply:
Good point, removed the exception and rephrased the description a bit.